tests/rptest/services/franz_go_verifiable_services.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2022 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import threading
+from ducktape.services.background_thread import BackgroundThreadService
+
+# The franz-go root directory
+TESTS_DIR = os.path.join("/opt", "si-verifier")
+
+from enum import Enum
+
+
+class ServiceStatus(Enum):
+    SETUP = 1
+    RUNNING = 2
+    FINISH = 3
+
+
+class FranzGoVerifiableService(BackgroundThreadService):
+    """
+    FranzGoVerifiableService is si-verifier service.
```

Comment:
why is the `si-verifier` named `FranzGoVeri...`? I don't think it needs to change, but I'm curious if it is hinting at a future generalization or something.

Reply:
It's naming that should converge at some point -- `si-verifier` is kind of a misleading name because there's nothing really shadow indexing specific about it, other than that the random read mode was designed to thrash the SI cache.

Reply:
Should we fork it to organization and rename. Or can do it later